pub/spec/fixtures/pub_dev_responses/simple/README.md

```diff
@@ -0,0 +1,8 @@
+Responses from the pub.dev website.
+
+These can be regenerated with:
+```bash
+for package in cli_util protobuf fixnum path retry; do
+  curl --compressed https://pub.dev/api/packages/$package > pub/spec/fixtures/pub_dev_responses/simple/$package.json
```

Comment:
```suggestion
  curl --compressed https://pub.dev/api/packages/$package | jq > pub/spec/fixtures/pub_dev_responses/simple/$package.json
```

Then you'll get some nice formatting... but it's important :D

Reply:
Then we'll depend on jq... not sure I want that.